File: program/find-sum-of-ap-series/find-sum-of-ap-series.py
# Program to find the sum of n-terms in an AP

# The input to the program are a,n,d
# a - first term
# n - number of terms
# d - common difference

a, n, d = list(map(int, input().split(" ")))

if n < 0:
    print("Error! Number of terms cannot be negative")
else:
    print((n / 2) * ((2 * a) + ((n - 1) * d)))

File: program/find-sum-of-ap-series/find_sum_of_ap_series.go
package main

import "fmt"

func main() {
	var a int
	fmt.Scanf("%d", &a)
	var n int
	fmt.Scanf("%d", &n)
	var d int
	fmt.Scanf("%d", &d)
	sum := (2*a + ((n - 1) * d)) * n
	fmt.Println(sum / 2)
}
